pyproject.toml

```diff
@@ -40,12 +40,13 @@ dependencies = [
     "tiktoken",
     "pillow",
     "h11>=0.16.0",
-    "python-multipart>=0.0.20",               # For fastapi Form
-    "uvicorn>=0.34.0",                        # server
-    "opentelemetry-sdk",                      # server
+    "python-multipart>=0.0.20", # For fastapi Form
+    "uvicorn>=0.34.0", # server
+    "opentelemetry-sdk", # server
     "opentelemetry-exporter-otlp-proto-http", # server
-    "aiosqlite>=0.21.0",                      # server - for metadata store
-    "asyncpg",                                # for metadata store
+    "aiosqlite>=0.21.0", # server - for metadata store
+    "asyncpg", # for metadata store
+    "mcp>=1.8.1",
```

Comment:
mcp is only needed if the agents API is served. This is not a strong dependency for the server to run.

Reply:
@leseb I updated my branch to not have the commit changing the dependencies, as you have fixed that in #2587 (thanks btw!)

Reply:
Ok now we need https://github.com/meta-llama/llama-stack/pull/2621, sorry I forgot to pin the MCP version and we get 1.3...